compiler/packages/babel-plugin-react-compiler/src/ReactiveScopes/MemoizeFbtOperandsInSameScope.ts

```diff
@@ -67,7 +67,10 @@ function visit(fn: HIRFunction, fbtValues: Set<IdentifierId>): void {
          * all `fbt` string literals in case they are used as a jsx tag.
          */
         fbtValues.add(lvalue.identifier.id);
-      } else if (value.kind === "LoadGlobal" && FBT_TAGS.has(value.name)) {
+      } else if (
+        value.kind === "LoadGlobal" &&
+        FBT_TAGS.has(value.binding.name)
```

Comment:
I'm guessing that we want to keep this logic as-is (regardless of imports)? fbt-transform logic seems pretty fragile, so it seems fine to be conservative

Reply:
yeah i'm leaving the fbt stuff alone, regardless of where fbt comes from. although the fbt plugin does require that you import it.